packages/web/src/scss/settings/_dictionaries.scss

```diff
@@ -6,43 +6,123 @@ $size-extended: list.join($size, (xsmall, xlarge));
 
 $action-colors: (
     primary: (
-        default: tokens.$action-primary-default,
-        text: tokens.$text-primary-inverted-default,
+        default-color: tokens.$text-primary-inverted-default,
```

Comment:
What about to somehow generated this?

Reply:
The problem is anomalies. `primary` has `default-background` set to `tokens.$action-primary-default`, but `secondary` has it set to `tokens.$background-interactive-default`. So there is a pattern, but not everywhere. Inverted has all backgrounds set to transparent.

Reply:
And generating the same properties and adding only specific ones to the generated map?

Reply:
refactored, see latest commit

Reply:
For the record, I already pointed out to the designer there are chances to improve the system of tokens. For example, `$action-secondary-default` could point to the same direct color as `$background-interactive-default` which (along with many other changes) would clean it up and make generating possible.